### DISCUSSION TITLE
middle

### DISCUSSION THREAD

**Discussion by pierre-oord** [orestbida/cookieconsent#117] [Q&A]
Hi,

In the documentation it says it is possible to middle center the consent dialog, but this does not seem to work. In the demo, this option is also not available. Is the documentation outdated?

**Comment by orestbida** [comment]
@pierre-oord 

The documentation is definitely lacking in some areas. At the moment the `middle` will be enabled only if `force_consent` is also enabled (A choice I made when I first added the option). This behaviour will change and will be allowed in future versions. Check out #95.

**Reply by orestbida** [reply]
@pierre-oord ,

fixed in v2.7.0
